apps/files/src/components/FileListFilter/FileListFilterType.vue

```diff
@@ -45,6 +45,10 @@ export default defineComponent({
 	},
 
 	props: {
+		preset: {
```

Comment:
```suggestion
		presets: {
```
We should use either singular or plural everywhere to minimize confusion